sphinx_copybutton/_static/test.js

```diff
@@ -1,5 +1,6 @@
 import test from 'ava';
 import { formatCopyText } from "./copybutton_funcs";
+import { filterText } from "./copybutton_funcs";
```

Comment:
```suggestion
import { formatCopyText, filterText } from "./copybutton_funcs";
```
Is this how an import should look?  My quick check says yes...